chain/store/store.go

```diff
@@ -1561,7 +1549,7 @@ func (cs *ChainStore) WalkSnapshot(ctx context.Context, ts *types.TipSet, inclRe
 }
 
 func (cs *ChainStore) Import(r io.Reader) (*types.TipSet, error) {
-	header, err := car.LoadCar(cs.Blockstore(), r)
+	header, err := car.LoadCar(cs.StateBlockstore(), r)
```

Comment:
I'd at least leave some comment here noting that when chain/state blockstores split, this should be taken care of